tests/test_content_loader.py

```diff
@@ -836,6 +844,19 @@ def test_get_data(self):
         ])
         assert section.get_data(form)['q4'] == ['']
 
+        # if we have one empty value
+        form = ImmutableMultiDict([
+            ('q5-0', '')
+        ])
+        assert section.get_data(form)['q5'] == ['']
```

Comment:
Not sure what we wanted, really, but this seemed a reasonable guess.

With [lists](https://github.com/alphagov/digitalmarketplace-utils/blob/master/tests/test_content_loader.py#L837), the rule we follow is that we always return a list with an empty string.
And with [single boolean questions](https://github.com/alphagov/digitalmarketplace-utils/blob/master/tests/test_content_loader.py#L807), we return whatever non-boolean value was passed in because the API will catch it.